clients/include/testing_trtri_strided_batched.hpp

```diff
@@ -0,0 +1,216 @@
+/* ************************************************************************
+ * Copyright 2018-2019 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+#include "cblas_interface.hpp"
+#include "flops.hpp"
+#include "near.hpp"
+#include "norm.hpp"
+#include "rocblas.hpp"
+#include "rocblas_init.hpp"
+#include "rocblas_math.hpp"
+#include "rocblas_random.hpp"
+#include "rocblas_test.hpp"
+#include "rocblas_vector.hpp"
+#include "unit.hpp"
+#include "utility.hpp"
+
+template <typename T>
+void testing_trtri_strided_batched(const Arguments& arg)
+{
+    rocblas_int N           = arg.N;
+    rocblas_int lda         = arg.lda;
+    rocblas_int batch_count = arg.batch_count;
+
+    rocblas_int bsa    = lda * N;
```

Comment:
May want to use stride_a argument

Reply:
> May want to use stride_a argument, and use rocblas_stride variable type.

If you do so, you'll need to define the `rocblas_stride` type in `rocblas_common.yaml` as a "YAML typedef" name.